provision/acc_provision/cloud_provision.py

```diff
@@ -169,12 +169,22 @@ def Run(self, flavor_opts, kube_yaml_gen_func):
         output_tar = self.args.output_tar
         operator_cr_output_file = self.args.aci_operator_cr
         gen(self.config, output_file, output_tar, operator_cr_output_file)
-        if self.args.flavor == "aks":
-            self.print_aks_setup()
-        elif self.args.flavor == "eks":
-            self.print_eks_setup()
-        else:
-            self.print_openshift_setup()
+        try:
+            if "test_run" not in self.args._asdict():
+                if self.args.flavor == "aks":
+                    self.print_aks_setup()
+                elif self.args.flavor == "eks":
+                    self.print_eks_setup()
+                else:
+                    self.print_openshift_setup()
+        except AttributeError:
+            if "test_run" not in vars(self.args):
+                if self.args.flavor == "aks":
+                    self.print_aks_setup()
+                elif self.args.flavor == "eks":
+                    self.print_eks_setup()
+                else:
+                    self.print_openshift_setup()
```

Comment:
There is unnecessary code repetition here. "test_run" is only ever going to be part of args when we are running tests, and in that case the prints should not be called, otherwise the prints should be called.

Reply:
Makes sense, thanks for the suggestion. I have fixed it and tested it on my VM it works smoothly now.